app/assets/javascripts/discourse/components/post-menu.js.es6

```diff
@@ -349,6 +349,25 @@ const PostMenuComponent = Ember.Component.extend(StringBuffer, {
     this.sendAction('toggleBookmark', post);
   },
 
+  // Wiki button
+  buttonForWiki(post) {
+    if (!post.get('can_wiki')) return;
+
+    let buttonClass = 'wiki',
+        tooltip = 'post.controls.wiki';
```

Comment:
heh actually I was just referring to the `tooltip` variable. `buttonClass` could be kept as it is. We dup alot more code now XD